include/crocoddyl/core/mathbase.hpp

```diff
@@ -25,6 +25,7 @@ struct MathBaseTpl {
   typedef Eigen::Matrix<Scalar, 6, 1> Vector6s;
   typedef Eigen::Matrix<Scalar, 3, 3> Matrix3s;
   typedef Eigen::Matrix<Scalar, 6, 6> Matrix6s;
+  typedef Eigen::Matrix<Scalar, 4, 6> Matrix46s;
```

Comment:
Don't add this here. Use the typedef inside the cost itself. This is too specific, and this naming is a bit weird.